spec/syntax.md

```diff
@@ -463,8 +463,9 @@ The characters `\` and `|` MUST be escaped as `\\` and `\|`.
 
 **_Unquoted_** literals have a much more restricted range that
 is intentionally close to the XML's [Nmtoken](https://www.w3.org/TR/xml/#NT-Nmtoken),
-with the restriction that it MUST NOT start with `-` or `:`,
-as those would conflict with _function_ start characters.
+with the restrictions that it MUST NOT start with `:`,
+and when starting with `-`, must be followed by a `.` or a digit.
+These restrictions are required not to conflict with _function_ start characters.
```

Comment:
This is now a run-on sentence. Perhaps:

```suggestion
is intentionally close to XML's [Nmtoken](https://www.w3.org/TR/xml/#NT-Nmtoken).
To make _unquoted_ literals distinct from _function_ names, a literal 
MUST be quoted if it begins with a `:` or if it begins with a `-` that is 
**not** followed by a `.` or a digit. Literals beginning with a `-` followed
by a `.` or a digit might be used to pass a negative number value to
a _function_ or as a _key_ for a selector.
```

@eemeli An example I didn't think of calling out previously is that unquoted literals are also used on the right hand side of an `option`. I think there are **_lots_** of places where a negative number would be passed there--that would be the most common use:

```
 {$foo :function option=-42}
```

Reply:
I edited your comment to get the line breaks right, but then still needed to apply the commit locally to get it to stick. I left out the last sentence, as it didn't seem necessary and was a bit confusing in listing only some of the places where a negative literal could hereby be used.